Framework/script/RepoCleaner/tests/test_1_per_hour.py

```diff
@@ -35,86 +21,64 @@ class Test1PerHour(unittest.TestCase):
     one_minute = 60000
 
     def setUp(self):
-        self.ccdb = Ccdb('http://ccdb-test.cern.ch:8080')
+        self.ccdb = Ccdb('http://128.142.249.62:8080') # ccdb-test but please use IP to avoid DNS alerts
```

Comment:
to avoid hard-coding it I presume ? 

I have to think about it. Depending on an env var to be able to run a test is not optimal. I'll think how to do it.

Reply:
Yes. It seems it is now in a bunch of files, it's easy to forget about some when replacing the value.

Reply:
I extracted the URL to a global variable defined in `test_utils`. This way there is only one place to update.